routing/pathfind.go

```diff
@@ -1237,34 +1237,61 @@ func findBlindedPaths(g Graph, target route.Vertex,
 		return features.HasFeature(lnwire.RouteBlindingOptional), nil
 	}
 
-	// This function will have some recursion. We will spin out from the
-	// target node & append edges to the paths until we reach various exit
-	// conditions such as: The maxHops number being reached or reaching
-	// a node that doesn't have any other edges - in that final case, the
-	// whole path should be ignored.
-	paths, _, err := processNodeForBlindedPath(
-		g, target, supportsRouteBlinding, nil, restrictions,
+	// Initialize the map with the chained channel list.
+	alreadyVisited := make(map[route.Vertex]bool,
+		len(partiallySpecifiedPath)+1,
```

Comment:
you got it on the wrong place:

here it needs to be just len(partiallySpecifiedPath)

but in `processNodeForBlindedPath` it needs to be:

```
visited := make(map[route.Vertex]bool, len(alreadyVisited)+1)
	for r := range alreadyVisited {
		visited[r] = true
	}
```